### DISCUSSION TITLE
Comparing causal graph and its ground-truth graph

### DISCUSSION THREAD

**Discussion by lcastri** [jakobrunge/tigramite#174] [Q&A] [answered]
Hi, I want to ask a question about how to compare two causal graphs. In particular, I have the ground-truth causal graph and the causal graph resulting from PCMCI algorithm. 
The two graphs have exactly the same structure in terms of links but they differ in terms of strength of the links. Having the true causal graph. is it possible to compare the one coming from PCMCI by some parameters from the output of the algorithm (e.g. p-value of the links)?

Thanks for reading and I appreciate any thoughts you have.

**Comment by jakobrunge** [comment, marked as answer]
That's up to you! I would *not* use p-values, though, but rather the MCI partial correlation value. You can then use some kind of mean squared error metric...
